src/ripple/app/main/Application.cpp

```diff
@@ -1182,9 +1180,15 @@ bool ApplicationImp::setup()
         if (logs_->threshold() > kDebug)
             logs_->threshold (kDebug);
     }
+    JLOG(m_journal.info()) << "process starting: "
+        << BuildInfo::getFullVersionString();
 
+    // Optionally turn off logging to console.
     logs_->silent (config_->silent());
 
+    // VFALCO NOTE: 0 means use heuristics to determine the thread count.
```

Comment:
Since you're here, this comment should just be removed from here.

Reply:
fixed